app/utils/testing/accessibility-checker.ts

```
@@ -0,0 +1,705 @@
+  try {
+    // 既知の色の組み合わせでテスト
+    const testCases = [
+      { color1: 'rgb(0, 0, 0)', color2: 'rgb(255, 255, 255)', expected: 21 },
+      { color1: '#000000', color2: '#ffffff', expected: 21 },
+      { color1: 'rgb(255, 255, 255)', color2: 'rgb(0, 0, 0)', expected: 21 },
+      { color1: 'rgb(128, 128, 128)', color2: 'rgb(255, 255, 255)', expected: 3.95 },
+    ];
+    
+    for (const testCase of testCases) {
+      const ratio = calculateContrastRatio(testCase.color1, testCase.color2);
+      if (Math.abs(ratio - testCase.expected) > 0.1) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getEffectiveBackgroundColor'. Consider to delete it or rename it.

'getEffectiveBackgroundColor' is defined here:


(lint/suspicious/noRedeclare)